ginga/pilw/ImageViewPil.py

```diff
@@ -4,19 +4,24 @@
 # This is open-source software licensed under a BSD license.
 # Please see the file LICENSE.txt for details.
 
-import numpy
 from io import BytesIO
 
-from PIL import Image
-from . import PilHelp
-from .CanvasRenderPil import CanvasRenderer
+import numpy
 
 from ginga import ImageView
 
+try:
+    from PIL import Image
+    from . import PilHelp  # noqa
+    from .CanvasRenderPil import CanvasRenderer
+except ImportError:  # Need this for Sphinx build
+    pass
```

Comment:
Hmm, I think a silent pass on `ImportError` is going to cause problems for someone trying to use some of the back ends when `pillow` is not installed.  I think `pillow` is used enough places that it might be ok just to make it a requirement.

Reply:
The thing is that it is unnecessary overhead to have RTD install `pillow` just to get this import to work. The documentation build itself does not need `pillow`. However, including `pillow` in RTD requirements does seem less risky than here, so sure, I can add that.

Reply:
Well, I'm concerned that having imports fail silently in a module that needs them is probably not good, so just trying to think of alternatives.  If this module is getting included as part of generating documentation then maybe we should just add the package that is used to the documentation build process, even if it is not actually used in the build.

Reply:
Yes, I agree. I did that in my latest commit. Thanks!